chia/cmds/wallet.py

```diff
@@ -167,8 +167,11 @@ def show_cmd(wallet_rpc_port: Optional[int], fingerprint: int, wallet_type: Opti
 )
 @click.option("-i", "--id", help="Id of the wallet to use", type=int, default=1, show_default=True, required=True)
 @click.option("-f", "--fingerprint", help="Set the fingerprint to specify which wallet to use", type=int)
-def get_address_cmd(wallet_rpc_port: Optional[int], id, fingerprint: int) -> None:
-    extra_params = {"id": id}
+@click.option(
+    "-n", "--new-address", help="Create a new wallet receive address", is_flag=True, default=False, show_default=True
```

Comment:
I like to make sure that either value can be selected for a flag.  This is valuable for scripting and backwards compatibility if we were to shift to a different default in the future.

I can't swear to the handling of this with both short and long names, but at least the `/` in the long name works.

```suggestion
    "-n/-l", "--new-address/--latest-address", help="Create a new wallet receive address", show_default=True
```

Reply:
Added the complementing option with an updated help string